lib/sda/overlay.go

```diff
@@ -28,17 +28,26 @@ type Overlay struct {
 	entityListLock sync.Mutex
 	// cache for relating token(~Node) to TreeNode
 	cache TreeNodeCache
+
+	// TreeNodeInstance part
+	instances     map[TokenID]*TreeNodeInstance
+	instancesInfo map[TokenID]bool
```

Comment:
It's not used **yet**. With Node, when a Node is finished, we want to be sure that we drop the messages destined to it that are coming after. Otherwise, the Overlay will just recreate a new Node and dispatch the message to it.
We may be thinking of an other solution, it was a quick hack during the dev of ByzCoin because we had this problem.